projects/material-addons/src/lib/data-table/data-table-filter/data-table-filter.module.ts

```diff
@@ -0,0 +1,12 @@
+import { NgModule } from "@angular/core";
+import { CommonModule } from "@angular/common";
+import { DataTableFilter } from "./data-table-filter.directive";
+import { DataTableFilterComponentModule } from "./data-table-filter-component/data-table-filter.component.module";
+import { DataTableFilterHeader } from "./data-table-filter-header.directive";
+
+@NgModule({
```

Comment:
is there a reason to use modules? we migrated the whole project to standalone 1 year ago. Can you please change all new components/directives to standalone?